### ISSUE TITLE
Migrate from Travis CI to GitHub Actions 

### ISSUE THREAD

**Issue by robbyoconnor** [nickjj/ansible-acme-sh#19]
@nickjj I hope you don't mind, I'm making an issue so that if someone wants to take it up, they can to alleviate some of your workload.

**Comment by nickjj** [comment]
Thank you, it shouldn't be too crazy to get it moved over. Most of the work is done in the `test.sh` script which is downloaded and run on each CI run.

**Comment by robbyoconnor** [comment]
yeah, I'll try to get to it but I also have a letsencrypt cert expiring and need to get acme.sh set up hence getting the role in a state that I can use.
